### PR TITLE
[vc help] add build command to help/--help

### PR DESCRIPTION
**Pull request by trek**
Not sure how we've missed this all these years!

### CODE REVIEW

File: packages/cli/src/args.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ export const help = () => `
       deploy               [path]      Performs a deployment ${chalk.bold(
         '(default)'
       )}
+      build                            Build the project locally
```
Comment (erikareads) [inline packages/cli/src/args.ts]: There will be a snapshot test that fails because of this change.

**Review by erikareads — CHANGES_REQUESTED** [review]

File: packages/cli/src/args.ts (hunk 1 above)
Comment (EndangeredMassa) [inline packages/cli/src/args.ts]: **suggestion (soft-blocking):** 

```suggestion
      build                            Build the project locally into `./vercel/output`
```

Comment (trek) [reply packages/cli/src/args.ts]: @erikareads I oddly couldn't find the snapshot (or the test even) for `help/--help` outside the context of a specific command. Any idea where to look?

Comment (trek) [reply packages/cli/src/args.ts]: I guess we'll see in a moment!

Comment (trek) [reply packages/cli/src/args.ts]: Aha! `test/unit/args.test.ts`. I was just looking at `help.test.ts`

**Review by erikareads — APPROVED** [review]

**Pull request merged**